phpstan.neon.dist

```diff
@@ -24,6 +24,7 @@ parameters:
 		- system/Commands/Generators/Views/*
 		- system/Config/Routes.php
 		- system/Debug/Toolbar/Views/toolbar.tpl.php
+		- system/Images/Handlers/GDHandler.php
```

Comment:
PHP 8 uses `GDImage` class type for the `image*` functions, which works fine with our current code but breaks the analysis. Ignoring the only file with these references until it can be reworked to support all versions or until PHPStan adds version-specific exemption handling.

Reply:
Maybe we can use stub files for this? I'm not sure if there is a capability in phpstan to conditionally include stub files based on the current PHP version.